tests/json_test.cpp

```diff
@@ -32,7 +30,7 @@ void JsonDefaultTest(const std::string& tests_data_path) {
   parser.opts.output_enum_identifiers = true;
   flatbuffers::FlatBufferBuilder builder;
   auto name = builder.CreateString("default_enum");
-  MonsterBuilder color_monster(builder);
+  MyGame::Example::MonsterBuilder color_monster(builder);
```

Comment:
how about shortening the namespace with `namespace me = ::MyGame::Example;` or something. This is a common technique my team uses without resorting to `using` statements.